### PR TITLE
ROS 2 Port

### PR DESCRIPTION
**Pull request by jhdcs**
This is a port of rqt_robot_monitor to ROS 2's Dashing version. It's been lightly tested, but appears to run correctly when paired with the ROS 2 port of Diagnostic Aggregator.

### CODE REVIEW

**Comment by arne48** [comment]
I could verify that the plugin is working under dashing, and would take @norro's word for foxy but will test that myself as well later.
Therefore I will prepare the release repositories for dashing, foxy, and maybe eloquent for completeness.
However, I hope that @norro or @Karsten1987 will also release a version of the diagnostics package which is compatible with dashing.
Currently, in their port PR, commit adf3011 breaks the timer API compatibility for a stock dashing installation.
Long story short, this plugin also is only useful for distributions, which have a working diagnostics(diagnostic_aggregator) package.
